### PR TITLE
[#277] Do not add unreadable files to sync chunks

### PR DESCRIPTION
**Pull request by alanking**
Addresses #277 

Confirmed that test fails before fix. Test suite is passing.

### CODE REVIEW

File: irods_capability_automated_ingest/test/test_irods_sync.py [hunk 1]
```diff
@@ -1911,6 +1911,43 @@ def test_exclude_file_type_single_file_link(self):
...
+            with open(unreadable_file_path, "w") as f: 
+                f.write("this will not be read!")
+            os.chmod(unreadable_file_path, 0o000)
+            # Run the sync job...
+            proc = subprocess.Popen(sync_cmd)
+            proc.wait()
+            workers = start_workers(1)
+            wait_for(workers, job_name)
+            # Make sure no jobs fail...
+            self.do_assert_failed_queue(count=None, job_name=job_name)
```
Comment (korydraughn) [inline irods_capability_automated_ingest/test/test_irods_sync.py:1943]: What does `count=None` do/mean?

**Review by korydraughn — APPROVED** [review]
Looks good to me.

If the tests pass, pound it.

File: irods_capability_automated_ingest/test/test_irods_sync.py (hunk 1 above)
Comment (alanking) [reply irods_capability_automated_ingest/test/test_irods_sync.py:1943]: It indicates that the count stored in Redis for the failed tasks key is expected to be `None`, meaning, no tasks failed, so the value was not set.

Comment (korydraughn) [reply irods_capability_automated_ingest/test/test_irods_sync.py:1943]: Got it.

**Comment by alanking** [comment]
#'d, mergin

**Pull request merged**